brvs-core/src/main/java/iroha/validation/transactions/core/provider/impl/RegisteredUsersStorageImpl.java

```diff
@@ -77,6 +77,11 @@ public boolean contains(String accountId) {
     return collection.find(eq(USER_NAME_ATTRIBUTE, accountId)).first() != null;
   }
 
+  @Override
+  public void remove(String accountId) {
+    collection.deleteOne(eq(USER_NAME_ATTRIBUTE, accountId));
```

Comment:
Is it a performance matter? Maybe we will try to remove a collection at once?

Reply:
Good point